health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/accessbilityPopUP.js

```
@@ -62,76 +70,90 @@ const AccessibilityPopUp = ({ onClose, census }) => {
+          </Card>,
+        ]}
+        onOverlayClick={onClose}
+        footerChildren={[
+          <Button
+            className={"campaign-type-alert-button"}
+            type={"button"}
+            size={"large"}
+            variation={"secondary"}
+            label={t(`HCM_MICROPLAN_VILLAGE_ACCESSIBILITY_CLOSE_LABEL`)}
+            onClick={onClose}
+          />,
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)